analysis/Analysis/Section_3_5.lean

```diff
@@ -291,7 +291,7 @@ theorem SetTheory.Set.Fin.toNat_lt {n:ℕ} (i: Fin n) : i < n := (toNat_spec i).
 theorem SetTheory.Set.Fin.coe_toNat {n:ℕ} (i: Fin n) : ((i:ℕ):Object) = (i:Object) := by
   set j := (i:ℕ); obtain ⟨ h, h':i = Fin_mk n j h ⟩ := toNat_spec i; rw [h']
 
-@[simp]
+@[simp low]
```

Comment:
This one was getting in the way. I'm not sure if it's better to remove it but for now I bumped its priority down. I'll think more about what to do here, but outside of this PR.